docs/DeveloperGuide.md

```diff
@@ -344,58 +359,56 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 3.
 
-**Use case: Listing out information of all residents**
+**Use case: UC3. Listing out information of all residents**
 
 **MSS**
-1. User wants to see the full list of residents in RC4.
-2. User uses the list command.
-3. RC4HDB displays the details of all residents in RC4.
+1. User wants to see the full list of residents in [**RC4**](#glossary).
+2. RC4HDB displays the details of all residents in [**RC4**](#glossary).
 
     Use case ends.
 
-**Extension**
+**Extensions**
 * 2a. The list is empty.
 
     Use case ends.
 
 
-**Use case: Editing a single resident’s information**
+**Use case: UC4. Editing a single resident’s information**
 
 **MSS**
-1. Resident has a change in personal information.
-2. User needs to edit the resident's information in RC4HDB.
-3. User looks at which category the information falls under.
-4. User uses the edit command to edit the resident information.
-5. RC4HDB updates the information of the specified resident.
-6. RC4HDB displays a message detailing the changes made.
+1. Resident has a specific change in personal information.
+2. User edits the resident's information in RC4HDB.
+3. RC4HDB updates the information of the specified resident.
+4. RC4HDB displays a message detailing the changes made.
 
     Use case ends.
 
 **Extensions**
 
-* 3a. There is no relevant category for that information.
-
+* 2a. There is no relevant category for that information.
+  * 2a1. RC4HDB shows an error message.
+  
     Use case ends.
 
 
-* 4a. User enters resident information in an invalid format.
+* 2b. User enters resident information in an invalid format.
 
-    * 4a1. RC4HDB shows an error message.
+    * 2b1. RC4HDB shows an error message.
 
-      Use case resumes at step 3.
+      Use case resumes at step 2.
 
 
-**Use case: Finding a resident’s information by their name**
+**Use case: UC 5. Finding a resident’s information by their name**
 
 **MSS**
 1. User wants to search for a resident’s information.
-2. User uses the find command to find the resident.
-3. RC4HDB searches the database for the user's input.
+2. User finds a resident by their name.
```

Comment:
Sounds a bit weird here, maybe change it to "**User makes a request to RC4HDB to find a resident by their name.**"